meta-resin-common/recipes-containers/mkfs-hostapp-native/files/create

```diff
@@ -4,11 +4,11 @@ set -ex
 
 SYSROOT="/mnt/sysroot/inactive"
 
-dockerd -s=@DOCKER_STORAGE@ --data-root="$SYSROOT/docker" -H unix:///var/run/docker-host.sock &
+balenad -s=@DOCKER_STORAGE@ --data-root="$SYSROOT/balena" -H unix:///var/run/balena-host.sock &
 pid=$!
 sleep 5
 
-hostapp-update -f /input
+hostapp-update -f /input -n
```

Comment:
Could we somehow make yocto actually run the hooks to create the image? This will give us extra confidence that they work as intended

Reply:
This would need a lot of work and I would vote for a separate PR in the future. It touches the image generation class and we need to make hooks run at build.